### PR TITLE
[proto] Fixed to_image_tensor and to_image_pil

### PR DESCRIPTION
**Pull request by vfdev-5**
- Fixed to_image_tensor and to_image_pil
- Replaced copy arg by mode in to_image_pil

### CODE REVIEW

File: torchvision/prototype/transforms/functional/_type_conversion.py [hunk 1]
```diff
@@ -27,20 +27,25 @@ def label_to_one_hot(label: torch.Tensor, *, num_categories: int) -> torch.Tenso
 
 
 def to_image_tensor(image: Union[torch.Tensor, PIL.Image.Image, np.ndarray], copy: bool = False) -> torch.Tensor:
+    if isinstance(image, np.ndarray):
+        image = torch.from_numpy(image)
```
Comment (datumbox) [inline torchvision/prototype/transforms/functional/_type_conversion.py:31]: This makes the kernel non-jit scriptable.

Comment (vfdev-5) [reply torchvision/prototype/transforms/functional/_type_conversion.py:31]: was this kernel scriptable before ?

Comment (datumbox) [reply torchvision/prototype/transforms/functional/_type_conversion.py:31]: Hmm I guess not because of the invocation of `pil_to_tensor`.

**Review by datumbox — APPROVED** [review]
LGTM!

@pmeier thoughts?

**Comment by vadimkantorov** [comment]
Related: https://github.com/pytorch/vision/issues/5297

Maybe at least add a test on empty PIL images? (if it doesn't exist)
